docs/journalist/introduction.rst

```diff
@@ -6,5 +6,84 @@ Introduction
+
+Virtual machines that run on your Mac, Windows, or Linux machine (such as those
+created using VirtualBox, Parallels, and so on) are a "guest" on your machine,
+but still require a "host" operating system on top of which to run. These types
+of virtual machines are not designed as security tools; if the host OS is
+compromised, there are no protections for the guest OS, and some features (such
+as networking) allow communications between guest and host that can compromise
+the security of both.
+
+In contrast, Qubes virtualization occurs at a lower level, under the `Xen
+hypervisor`_. This means that virtual machines (VMs) in a Qubes environment
+can run operating systems that are isolated from and independent of each
+other. In addition, virtual machines can be used to quarantine specific
+functions of your computer. For example, network access is provided via two or
+more VMs, and you can control which applications or files
+have access to a networked environment by connecting to or disconnecting from
+these VMs.
+
+Finally, Qubes is designed to make it more difficult for malware to remain on
+your machine. Each VM has read-only access to the root filesystem that
+provides its operating system, meaning that if a VM is infected
+with malware, it will be more difficult for that malware to persist across a
+reboot of that VM.
+
+For more about the security features of Qubes, see
+`the Qubes OS documentation`_.
+
+.. _`Xen hypervisor`: https://wiki.xen.org/wiki/Xen_Project_Software_Overview
+.. _`the Qubes OS documentation`: https://www.qubes-os.org/faq/#general--security
+
 What is SecureDrop Workstation?
 -------------------------------
+
+SecureDrop Workstation is a pilot project that uses Qubes to make
+SecureDrop faster and simpler for journalists to use.
+
+A key feature of SecureDrop is that journalists can receive submissions from
+unknown sources without risking the security of their own machines and
+networks. Previously, SecureDrop accomplished this by using a physical airgap
+(the *Secure Viewing Station*), meaning that to view submissions, journalists
+would have to download them, transfer them to an encrypted USB drive, and
+physically take that drive to a separate, non-networked computer for decryption
+and viewing. SecureDrop Workstation combines all of those steps
+into one workflow on one machine: a Qubes computer that
+combines the *Journalist Workstation* and the *Secure Viewing Station*.
+
+How does it work?
+~~~~~~~~~~~~~~~~~
+
+SecureDrop Workstation contains several different carefully-configured qubes,
+so that everything a journalist needs to use SecureDrop resides on one
+computer. Encryption and decryption happen with one click using a network-
+isolated qube that holds the SecureDrop Submission Key. Submissions can be
+viewed securely on the same machine thanks to a `feature of Qubes`_ that
+creates temporary containers in which to view untrusted content without
```

Comment:
Since "container" is a technical term with a specific meaning, I feel a bit iffy about also using it as a nontechnical term to explain Qubes. There's really no way around running into "VMs" sooner or later when using Qubes, so IMO this is a case where it's better to use the precise technical term throughout. Alternatively, we could use the "qube" term Qubes itself uses a bit more.

Reply:
changed to "VM" or virtual machines in 5eda33f

Reply:
:heavy_check_mark: